httpx/middleware/newrelic_tracer.go

```diff
@@ -0,0 +1,59 @@
+package middleware
+
+import (
+	"fmt"
+	"github.com/remind101/nra"
+	"github.com/remind101/pkg/httpx"
+	"golang.org/x/net/context"
+	"net/http"
+)
+
+type NewRelicTracer struct {
+	handler  httpx.Handler
+	tracer   nra.TxTracer
+	router   *httpx.Router
+	createTx func(string, string, nra.TxTracer) nra.Tx
+}
+
+func NewRelicTracing(h httpx.Handler, router *httpx.Router, tracer nra.TxTracer) *NewRelicTracer {
+	return &NewRelicTracer{h, tracer, router, createTx}
+}
+
+func (h *NewRelicTracer) ServeHTTPContext(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
+	templatePath := h.templatePath(r)
+	transactionName := fmt.Sprintf("%s (%s)", templatePath, r.Method)
```

Comment:
What about `METHOD /path` or `METHOD: /path`? I think that's what we use in API. Feels a little more consistent with the http spec.